srcpkgs/arc-theme/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'arc-theme'
+pkgname=arc-theme
+version=20180114
+revision=1
+noarch=yes
+build_style=gnu-configure
+configure_script="./autogen.sh"
+hostmakedepends="automake pkg-config"
+makedepends="gtk+3-devel sassc"
```

Comment:
`sassc` ? what is it?

Reply:
It's a dependency required by the [new upstream](https://github.com/NicoHood/arc-theme/#manual-installation) - see [sassc](https://github.com/sass/sassc). The template for `sassc` doesn't restrict the architecture that it builds for so I'm unsure why it's unavailable for testing on `aarch64` and `armv7l`.

Reply:
ok, I see...

https://travis-ci.org/void-linux/void-packages/jobs/398116820#L921

Try to put `sassc` in the `hostmakedepends` instead?